### PR TITLE
Added jailbreak types and methods

### PR DESCRIPTION
**Pull request by solid8447**

### CODE REVIEW

**Comment by dr-nefardio3058** [comment]
Hello there! Thank you for submitting your pull request! I wanted to let you know that your changes look fantastic, and I can see that you put a lot of effort into them. Your addition of the new feature to the codebase will help the project be even more successful, and we're grateful for your contribution!

I have reviewed your code, and everything seems to be in order. Your changes will undoubtedly benefit the project, and I believe that they are ready to merge. 

Once again, we appreciate your hard work, and we're excited to see what other fantastic ideas you have to offer. Keep up the excellent work!

🎉🎉🎉Thank you for being part of our community and contributing to this project!🎉🎉😄

File: docs/prompt_hacking/injection.md [hunk 1]
```diff
@@ -36,6 +36,29 @@ The reason this works is that `remoteli.io` takes a user's tweet and concatenate
...
 any text the Twitter user injects into their tweet will be passed into the LLM.
 
+## Token Smuggling
+
+The term token smuggling refers to supplying the model with otherwise banned words. If you try to enter any of these words in the prompt directly, the model will not work. If you use code, and pass the words in pieces. When the model reconstructs the code from the previous prompt, it forces the model to complete the sentence. We can again mask it as a coding task (called autoregressive generation), and fool it into generating a sample output of the code, where it is originally working on malicious prompt. The smuggled tokens can virtually model to create any scenario - (@Ninkat2023TokensmugglingDef)(@vaibhavk972023Tokensmuggling)(@Ninkat2023TokensmugglingDef)
+
+### Masked Language Modeling
+
+The task of masking tokens in a sequence with a masking token and directing the model to fill that mask with an appropriate token is known as masked language modeling. This allows the model to focus on both the right and left contexts (tokens on the right side of the mask) (tokens on the left of the mask).
```
Comment (trigaten) [inline docs/prompt_hacking/injection.md:45]: Why is this here? Same question with Autoregressive language modeling and Autoencoder (AE) Language Model

File: docs/prompt_hacking/jailbreaking.md [hunk 2]
```diff
@@ -13,6 +13,25 @@ features to ensure that their models do not produce controversial (violent, sexu
 responses(@markov_2022)(@openai_api). This page discusses jailbreaks with ChatGPT (an OpenAI model), which has known difficulties deciding whether to reject harmful prompts (@openai_chatgpt). Prompts that successfully jailbreak the model often provide context
 for certain scenarios that the model has not been trained against.
 
+Integrated jailbreak:
```
Comment (trigaten) [inline docs/prompt_hacking/jailbreaking.md:16]: Can you provide examples of all of these?

**Comment by trigaten** [comment]
Please fix merge conflicts.
